src/main/docs/guide/lambda/requestHandlers.adoc

```diff
@@ -45,3 +45,11 @@ With api:function.aws.MicronautRequestHandler[], it is expected that you supply
 | Raw streams
 | api:function.aws.MicronautRequestStreamHandler[]
 |===
+
+AWS Lambda runtime will reuse the instances of your handler classes but the Java Virtual Machine will be suspended between the invocation. Every call on a single instance of your handler will reuse the same application context. If you need to perform actions before the JVM gets suspended then you can create **synchronous** handler for `AfterExecutionEvent` event which is published right before the execution of the method `handleRequest` in `MicronautRequestHandler` and `MicronautRequestStreamHandler` is finished.
```

Comment:
Create a new asciidoc file and add it to the `toc.yml`. So that it is easy to link to.

Reply:
resolved via https://github.com/micronaut-projects/micronaut-aws/pull/1433/commits/a9f862da9182c451fc7466c4145595e4e8e3689f